.github/workflows/linting.yml

```diff
@@ -4,10 +4,14 @@ name: Linting
 on:
   pull_request:
 
+permissions: {}
+
 jobs:
   apply-suggestions-commits:
     name: 'No "Apply suggestions from code review" Commits'
     runs-on: ubuntu-latest
+    permissions:
+      actions: none
```

Comment:
Are these necessary given that you specified empty permissions for the entire workflows?